src/views/design/ScriptDetail/index.tsx

```diff
@@ -312,6 +318,11 @@ const ScriptDetail = withStyles(styles)(
                                 value={newScriptDetail && newScriptDetail.description
                                     ? newScriptDetail.description : ''}
                                 onChange={(e) => this.updateScript({ description: e.target.value })}
+                                InputProps={{
+                                    readOnly: !checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC'),
```

Comment:
Can we already use the security group name of the script in question instead of the fixed value 'PUBLIC'.

Reply:
Also, what happens when creating a new script?